README.md

```diff
@@ -39,21 +39,21 @@ django-admin startproject theprojectname --extension py,yml,json --name Procfile
 - [ ] `npm update --save-dev`
 - [ ] Check for outdated npm dependencies with `npm outdated` and update them.
 - [ ] Change the first line of README to the name of the project.
-- [ ] Add an email address to the `ADMINS` settings variable in `theprojectname/theprojectname/settings/base.py`
+- [ ] Add an email address to the `ADMINS` settings variable in `{{theprojectname}}/{{theprojectname}}/settings/base.py`
 - [ ] Change the `SERVER_EMAIL` to the email address used to send e-mails.
 
 After completing ALL of the above, remove this `Project bootstrap` section from the project README. Then follow `Running` below.
 
 ## Running
 ### Setup
 - On project root, do the following:
-- Create a copy of ``theprojectname/settings/local.py.example``:  
-  `cp theprojectname/settings/local.py.example theprojectname/settings/local.py` (don't forget to replace `theprojectname` with your project's name again!).
+- Create a copy of ``{{theprojectname}}/settings/local.py.example``:  
+  `cp {{theprojectname}}/settings/local.py.example {{theprojectname}}/settings/local.py` (remembering you should remove the curly braces and replace `{{theprojectname}}` with your project's name!).
```

Comment:
I think the "remove the curly braces" is not necessary, since you're already saying that the person should replace `{{theprojectname}}` (which includes the curly braces) :)